composer.json

```diff
@@ -21,7 +21,7 @@
         "source": "https://github.com/cakephp/acl"
     },
     "require": {
-        "php": "^7.2",
+        "php": ">=7.4.0",
         "cakephp/cakephp": "^4.0",
```

Comment:
Should we pin this to `^4.3` ?

Reply:
@markstory  You mean `cakephp/cakephp ^4.3` ?

Reply:
The only reason I didn't was because only the tests need CakePHP 4.3, but that's really a decision for you guys

Reply:
Doesn't changing the min PHP version effectively bump up the cake version as well though?

Reply:
I've rebased the PHP version to `>=7.2` so we don't force a cakephp version bump